examples/Makefile.am

```diff
@@ -35,6 +35,7 @@ endif
 if HAVE_AUDIOTOOLBOX
 uplay_LDADD += $(UPIPEOSX_LIBS)
 endif
+uplay_CFLAGS = $(AVUTIL_CFLAGS)
```

Comment:
Should be $(UPIPEAV_CFLAGS) for consistency.

Reply:
Do you want me to do UPIPEAV_CFLAGS = $(AVUTIL_CFLAGS) ?

Reply:
UPIPEAV_CFLAGS should already exist.

Reply:
It seems there is no way to use `libupipe_av_la_CFLAGS` in examples/Makefile.am, so your suggestion (`UPIPEAV_CFLAGS = $(AVUTIL_CFLAGS)`) looks fine to me.